docs/sql_reference/src/asciidoc/_chapters/sql_functions_and_expressions.adoc

```diff
@@ -3638,6 +3776,52 @@ The example displays only part of the result table of the EXPLAIN
 function. It first uses the EXPLAIN function to generate the table and
 then selects the desired columns.
 
+<<<
+[[externaltolob_function]]
+== EXTERNALTOLOB Function
+
+[[externaltolob_function_syntax]]
+=== Syntax Descriptions of EXTERNALTOLOB Function
+
+The EXTERNALTOLOB function converts data from an external file into LOB format. This function can be used in INSERT or UPDATE statement.          
```

Comment:
"an INSERT or UPDATE statement" or "INSERT or UPDATE statements" sounds a bit better

Reply:
"an INSERT or UPDATE statement" is preferable, thanks Dave. :)